pages/add-event/index.js

```diff
@@ -0,0 +1,22 @@
+import EventForm from "@/components/EventForm";
+import Link from "next/link";
+import { uid } from "uid";
+import { useRouter } from "next/router";
+
+export default function MyEvents({ ownEvents, setOwnEvents }) {
+  const router = useRouter();
+  function handleAddEvent(newEvent) {
+    setOwnEvents([...ownEvents, { id: uid(), ...newEvent }]);
+    alert("🎉You have added your Event successfully!");
+    router.push("/my-events");
+  }
```

Comment:
You actually have the same function already in your app component and pass it down as a prop. You can simply accept it in this component as a prop instead of writing this function again.

The last two lines (with the `alert` and `router.push`) you can add to your submit function in your form component, right after you call `onAddEvent(data)`.

```suggestion

```